apps/comments/lib/Search/CommentsSearchProvider.php

```diff
@@ -77,7 +77,7 @@ public function getName(): string {
 	/**
 	 * @inheritDoc
 	 */
-	public function getOrder(): int {
+	public function getOrder($from): int {
```

Comment:
```suggestion
	public function getOrder(string $from): int {
```

same on all other implementations